configure.ac

```diff
@@ -7197,11 +7196,8 @@ dnl By default we always compile these even when OpenSSL is available
 dnl (issue #14693). The modules are small.
 PY_STDLIB_MOD([_md5], [test "$with_builtin_md5" = yes])
 PY_STDLIB_MOD([_sha1], [test "$with_builtin_sha1" = yes])
-PY_STDLIB_MOD([_sha256],
-  [test "$with_builtin_sha256" = yes], [],
-  [-I\$(srcdir)/Modules/_hacl/include -I\$(srcdir)/Modules/_hacl/internal -D_BSD_SOURCE -D_DEFAULT_SOURCE])
-PY_STDLIB_MOD([_sha512],
-  [test "$with_builtin_sha512" = yes], [],
+PY_STDLIB_MOD([_sha2],
+  [test "$with_builtin_sha2" = yes], [],
```

Comment:
Nit: these lines can now be joined :)

Reply:
other things seem to keep them on multiple lines wen adding compiler or linker flags so I'll leave it this way.  m4 "style" is purely "be consistent with what I see nearby" for me. ;)